tests/namespaced.rs

```diff
@@ -1740,12 +1872,13 @@ mod namespaced {
         Ok(BODY.len() * 2)
     }
 
-    async fn hbone_read_write_stream(stream: &mut TcpStream) {
+    async fn hbone_read_write_stream(stream: &mut TcpStream) -> anyhow::Result<()> {
         const BODY: &[u8] = b"hello world";
-        stream.write_all(BODY).await.unwrap();
+        stream.write_all(BODY).await?;
         let mut buf = [0; BODY.len() + WAYPOINT_MESSAGE.len()];
-        stream.read_exact(&mut buf).await.unwrap();
+        stream.read_exact(&mut buf).await?;
```

Comment:
why? We want these to panic on errors so we assure tests fail if the r/w fail. Arguably updating to an expect could be helpful for locating where a test failure occurred.